pkg/agent/secondarynetwork/init.go

```diff
@@ -55,6 +61,31 @@ func Initialize(
 		return err
 	}
 
+	var addPrimaryInterfaceToBridge bool
+	for _, phyInterface := range config.OVSBridges[0].PhysicalInterfaces {
+		if phyInterface == primaryInterfaceName {
+			addPrimaryInterfaceToBridge = true
+			break
+		}
+	}
+
+	if addPrimaryInterfaceToBridge {
+		klog.InfoS("Physical interface is the primary interface, moving primary ips and routes to secondary bridge")
+		primaryInterface, primaryInterfaceIPs, primaryInterfaceRoutes, err := util.GetInterfaceConfig(primaryInterfaceName)
+		if err != nil {
+			return fmt.Errorf("failed to get primary interface configuration: %v", err)
+		}
+		if err = movePrimaryInterfaceConfigToBridge(ovsBridgeClient, primaryInterface, primaryInterfaceIPs, primaryInterfaceRoutes); err != nil {
+			return fmt.Errorf("failed to move primary interface config to secondary bridge: %v", err)
+		}
+
+		go func() {
```

Comment:
Maybe better to do this cleanup in `cmd/antrea-agent/agent.go`, after secondarynetwork.Initialize()